gempy/core/data.py

```diff
@@ -53,9 +53,9 @@ def __init__(self, project_name='default_project'):
 class Grid(object):
     """ Class to generate grids.
 
-    This class is used to create points where to
-    evaluate the geological model. This class serves a container which transmit the XYZ coordinates to the
-    interpolator. There are several type of grids objects will feed into the Grid class
+    This class is used to create points usede to
```

Comment:
@RichardScottOZ could you change your correction here please. I would then merge it! Thanks :)

Reply:
oops, ok!